oncue.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rspec_junit_formatter', '~> 0.1.5'
 
   gem.add_runtime_dependency 'addressable', '~> 2.3.3'
-  gem.add_runtime_dependency 'json', '~> 1.7.7'
-  gem.add_runtime_dependency 'rest-client', '~> 1.6.7'
+  gem.add_runtime_dependency 'json', '>= 1.7.7'
```

Comment:
@barrkel-m20  the changes have been applied in the newest commit. Please approve the change